test/Mono.Linker.Tests.Cases/Inheritance.Interfaces/OnReferenceType/NoKeptCtor/UnusedTypeHasExplicitInterfacePropertyPreservedViaXml.cs

```diff
@@ -1,6 +1,8 @@
 using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
 
 namespace Mono.Linker.Tests.Cases.Inheritance.Interfaces.OnReferenceType.NoKeptCtor {
+	[SetupLinkerArgument ("--disable-opt", "-unreachablebodies")]
```

Comment:
I don't love this double negative way of turning the option back on.  I was thinking a new `enable-opt` command line flag would be more intuitive to use.  That said, this is hopefully a one off temporary situation so I think it's OK.

Reply:
I agree and I'm not a fan of double negation either. I'd suggest the same as @mrvoorhe. Introduce `-enable-opt` as an undocumented option for now and use it in the tests.

Reply:
done